packages/terra-form-select/tests/jest/__snapshots__/Menu.test.jsx.snap

```diff
@@ -137,6 +136,10 @@ exports[`Menu correctly applies the theme context className 1`] = `
 </ThemeContextProvider>
 `;
 
+exports[`Menu should call focus on select component when select is closed via key event while not using safari browser 1`] = `null`;
```

Comment:
Why is this returning null?

Reply:
As per the discussion with @supreethmr and @saket2403 we decided to go only with wdio tests and revert the changes for jest.